cli/flox-rust-sdk/src/utils/mod.rs

```diff
@@ -96,3 +96,26 @@ pub fn copy_file_without_permissions(
     })?;
     Ok(())
 }
+
+/// Returns the last component of a dot-separated attribute path or the
+/// attribute provided if there is only one path component.
+pub fn attr_name(path: &str) -> String {
+    path.split('.')
+        .rev()
+        .next()
+        .map(|s| s.to_string())
+        .unwrap_or(path.to_string())
+}
```

Comment:
Is that toml paths or nix attr paths?

If it's nix you can parse it with the AttrPath structure we have in runix

Reply:
That's what it was for, but I ended up not using it. Once I had to start parsing descriptors I decided to leverage the parser I wrote in `pkgdb`, which already does that quote/dot-aware splitting of path components. I just forgot to remove this.